### ISSUE TITLE
Possibility to choose a music folders instead of scanning the whole system.

### ISSUE THREAD

**Issue by TClockBroughtMeHere** [Taiko2k/TauonMusicBox#1061]
Hello, I'm running Tauon 7.7.0, the Flatpak version on Linux Mint.

I would like to have the possibility to exclude folders, because at the moment, Tauon seems to scan my whole system (Linux Mint 21.1.). I have a workflow where I convert my FLACS to MP3s so I can transfer the smaller files to my phone. For that end, I create a folder called "Converted files" and add that to my system's main Music folder. I don't want these folders to mix, because I only listen to FLACS on my computer. 

I apologise if this feature has already been implemented, but if it exists, I couldn't find it. 

To recap, I want the ability to choose one music folder, an exclude any others.

**Comment by Taiko2k** [comment]
In this case just drag and drop that folder from your file manager into Tauon. Don't use the "import music folder" option that's in the main menu. (Which is what I assume you were using)

**Comment by TClockBroughtMeHere** [comment]
> In this case, create a new playlist in Tauon then just drag and drop that folder from your file manager into Tauon. Don't use the "import music folder" option that's in the main menu. (Which is what I assume you were using)
> 
> Basically Tauon doesn't have a global "folders to scan" setting, rather each playlist remembers the folders that were added to it individually.

Okay, I just "Cleared" the playlists and folders that I had. I was left with an empty interface. But when I add drag and drop my main music folder, it still displays the other folders when I start typing the artist name. It shows displays the songs twice, the other entries have grey text, but they still point to the same folder. And it also shows a folder "Converted files". It shows them as MP3s, although they are FLACs. I did convert those files to MP3, but they aren't even on my system any more.
